application/console/config/main.php

```diff
@@ -2,23 +2,8 @@
 return [
     'id' => 'app-console',
     'basePath' => dirname(__DIR__),
-    'bootstrap' => ['log', 'gii'],
-    'controllerNamespace' => 'console\controllers',
```

Comment:
yep, you're right @fillup , I did end up needing `gii` for the basemodels.  I'll bring all the necessary config's back in as we need them, e.g., `CronController`